sass/themes/frost/components/_progress_indicator.scss

```diff
@@ -0,0 +1,125 @@
+// ==========================================================================
+// Progress indicator
+// ==========================================================================
+
+$circle-size: 15px !default;
+$circle-line-thickness: 3px !default;
+$indicator-thickness: 2px !default;
+
+.flexer {
```

Comment:
@KoalaMango you could use `%placeholder` here to produce less CSS. Also Andy is working on adding the postcss for vendor prefixes which would make it unnecessary.